packages/user/src/components/Invitation/InvitationForm.tsx

```diff
@@ -1,11 +1,12 @@
-import { Provider, emailSchema } from "@dzangolab/react-form";
+import { Provider, emailSchema, useFormContext } from "@dzangolab/react-form";
 import { useTranslation } from "@dzangolab/react-i18n";
 import React, { useCallback, useState } from "react";
 import { toast } from "react-toastify";
 import * as zod from "zod";
 
 import { addInvitation } from "@/api/invitation";
 import { useConfig } from "@/hooks";
+import { AdditionalInvitationFields } from "@/types/invitation";
```

Comment:
@Nabin99 Could we export this from @types/indexts so that all types are imported from "@/types"
Same comment for this import in other files as well.